.github/workflows/workflow.yaml

```diff
@@ -305,11 +304,49 @@ jobs:
         run: |
           make e2e-verify-release IMG=${{ env.IMAGE_REPO }}:${TAG} USE_LOCAL_IMG=false
 
+      - name: Build gator-cli
+        run: |
+          build() {
+            export GOOS="$(echo ${1} | cut -d '-' -f 1)"
+            export GOARCH="$(echo ${1} | cut -d '-' -f 2)"
+            # in case of windows, we need to append .exe to the file name
+            EXTENSION="$(echo ${1} | cut -d '-' -f 3)"
+            EXTENSION=${EXTENSION/exe/.exe}
+            # build the binary
+            make bin/gator-${GOOS}-${GOARCH}${EXTENSION}
+            # rename the binary to gator
+            tmp_dir=$(mktemp -d)
+            cp bin/gator-${GOOS}-${GOARCH}${EXTENSION} ${tmp_dir}/gator${EXTENSION}
+            pushd ${tmp_dir}
```

Comment:
A way to make this simpler: skip the `pushd`, `popd` and move the `rm` command into a `trap`.  Like so:

```bash
tmp_dir=$(mktemp -d)
trap 'rm -rf -- "${tmp_dir}"' EXIT
```

All that aside... Isn't this environment ephemeral?  Perhaps we don't need to clean up at all?